src/components/SettingsModal.tsx

```
@@ -189,6 +190,7 @@
               </p>
               <Input
                 {...form.register('batchSize', { valueAsNumber: true })}
+                id='batchSize'
```

ESLint

[error] Unexpected usage of singlequote.

(jsx-quotes)